schematool/handlers_extra_test.go

```
@@ -0,0 +1,106 @@
+	}
+	body := w.Body.String()
+	if !strings.Contains(body, "[]") {
+		t.Errorf("expected empty list body, got %q", body)
+	}
+}
+
+func TestListSchemaDefinitionsHandler_WithExistingDir(t *testing.T) {
+	dir := t.TempDir()
+	withSchemaDir(t, dir)
+	// Create some sample schema files
+	os.WriteFile(filepath.Join(dir, "user.json"), []byte(`{"entityName":"user","fields":[]}`), 0644)
```

golangci-lint (2.11.4)

[error] Error return value of `os.WriteFile` is not checked

(errcheck)